README.md

```diff
@@ -67,7 +107,7 @@ defer {
     try? httpClient.syncShutdown()
 }
 
-var request = try HTTPClient.Request(url: "https://swift.org", method: .POST)
+var request = try HTTPClient.Request(url: "https://apple.com/", method: .POST)
```

Comment:
Why was this one changed?

Reply:
swift.org doesn't respect that we don't support compression by default which results in gibberish responses